src/App.js

```diff
@@ -105,6 +108,13 @@ export default {
 					`@font-face { font-family: 'wakamai-fondue'; src: url('${objectURL}'); }`
 				)
 			);
+		},
+		closeModal(forceClose) {
```

Comment:
I think it would make more sense to call this `toggleModal`. To make it the boolean argument more explicit perhaps you could use a named argument:

```js
toggleModal({ forceClose: false })
```

Or just use two separate functions altogether to prevent any confusion.